eng/testing/tests.mobile.targets

```diff
@@ -162,6 +162,23 @@
       AssemblySearchPaths="@(AssemblySearchPaths)" />
   </Target>
 
+  <UsingTask TaskName="TimeZoneLoader"
+             AssemblyFile="$(WasmBundleTaskTasksAssemblyPath)" />
+  <Target Condition="'$(TargetOS)' == 'Browser'" Name="LoadTimeZone" >
+    <TimeZoneLoader
+      InputDirectory="$(BundleDir)obj/data/input"
+      OutputDirectory="$(BundleDir)obj/data/output"
+      Version="2020a"/>
```

Comment:
Maybe this should be made a property? `$(TimeZoneDataVersion)`? Or whatever this version is.